apis/core/bootstrap/shapes/cube-project.ts

```diff
@@ -0,0 +1,30 @@
+import { cc } from '@cube-creator/core/namespace'
+import { hydra, rdfs, sh } from '@tpluscode/rdf-ns-builders'
+import { turtle } from '@tpluscode/rdf-string'
+
+export const CubeProjectShape = turtle`
+<shape/cube-project/create> {
+  <shape/cube-project/create> a ${sh.NodeShape}, ${hydra.Resource} ;
+    ${sh.targetClass} ${cc.CubeProject} ;
+    ${rdfs.label} "Cube Project" ;
+    ${sh.property} [
+      ${sh.name} "Project name" ;
+      ${sh.path} ${rdfs.label} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.order} 0 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Project source" ;
+      ${sh.path} ${cc.projectSourceKind} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.in} (
+        "CSV"
+        "Existing cube"
+      ) ;
+      ${sh.order} 1 ;
+    ] ;
+  .
+}
+`
```

Comment:
`sh:targetClass` is required to get validation running. It should match the `hydra:expects` annotation on the operation